back/src/Model/Pusher.ts

```diff
@@ -0,0 +1,42 @@
+import { SpaceSocket } from "../SpaceManager";
+import {
+    AddSpaceUserMessage,
+    BackToPusherSpaceMessage,
+    RemoveSpaceUserMessage,
+    UpdateSpaceUserMessage,
+} from "../Messages/generated/messages_pb";
+
+export type SpaceMessage = AddSpaceUserMessage | UpdateSpaceUserMessage | RemoveSpaceUserMessage;
+
+export class Pusher {
```

Comment:
Maybe try to find another name for this class? Is this supposed to represent a socket between pusher and back for the space messages?